pkg/api/dtos/alerting.go

```diff
@@ -64,3 +64,8 @@ type PauseAlertCommand struct {
 	AlertId int64 `json:"alertId"`
 	Paused  bool  `json:"paused"`
 }
+
+type PauseAlertsCommand struct {
+	DataSourceIds []int64 `json:"datasourceId"`
```

Comment:
I am not sure we need data source ids here, this is starting to become very similar to the silence rules mechanic we have talked about (were we want silence rules with filters like data source, alert name regex, alert tags etc). The pause all alerts is more a temporary measure until we have a good silence mechanic and I was thinking of keeping it simple for that reason.